contracts/deposit-contract/sendDepositTx/sendDeposits_test.go

```diff
@@ -134,28 +88,27 @@ func TestEndtoEndDeposits(t *testing.T) {
 
 	query := ethereum.FilterQuery{
 		Addresses: []common.Address{
-			web3Service.depositContractAddress,
+			testAcc.ContractAddr,
 		},
 	}
 
-	logs, err := testAcc.Backend.FilterLogs(web3Service.ctx, query)
+	logs, err := testAcc.Backend.FilterLogs(context.Background(), query) //context backgrround ?
```

Comment:
I am not sure if i am allowed to use testAcc.Backend.FilterLogs(context.Background() with context.Background()

Reply:
its the same type, it will be fine